pyInterface/bindings/partialWaveFit/fitResult_py.cc

```diff
@@ -304,6 +333,42 @@ namespace {
 		return self.Write(name);
 	}
 
+	bp::dict fitResult_getFitResultsFromFilesInMutibins(bp::list& fileNames, bp::str& treeName,
+	                                                    bp::str& branchName,
+	                                                    bp::object& onlyBestInMultibin, bp::object& stripMatricesFromFurtherAttempts,
+	                                                    bp::object& onlyConvergedResults) {
+		std::vector<std::string> fileNamesC;
+		rpwa::py::convertBPObjectToVector(fileNames, fileNamesC);
+
+		bp::dict fitResultsInMultibins;
+		std::map<rpwa::binningMapType, std::list<rpwa::fitResult> > fitResultsInMultibinsC =
+		        rpwa::getFitResultsFromFilesInMultibins(fileNamesC,
+		                bp::extract<std::string>(treeName),
+		                bp::extract<std::string>(branchName),
+		                bp::extract<bool>(onlyBestInMultibin),
+		                bp::extract<bool>(stripMatricesFromFurtherAttempts),
+		                bp::extract<bool>(onlyConvergedResults));
+
+		bp::object mod = bp::import("pyRootPwa.utils");
+
+		for (auto& binninbMap_fitResultsC : fitResultsInMultibinsC) {
+			bp::list results;
+			for (auto& fitResult : binninbMap_fitResultsC.second) {
```

Comment:
remove the element from `binninbMap_fitResultsC.second` after adding it to `results` to avoid keeping the fit results twice in memory.